### ISSUE TITLE
Fit Screen Ability?

### ISSUE THREAD

**Issue by Trickyhicky** [EDCD/EDMarketConnector#2172]
I run with three screens my main monitor and two other screens through SpaceDesk, 1x Samsung Pad, 1 x  Surface tablet.  

Currently each of the other displays a particular Add-On.  I have an old Mobile Smart Phone which would be ideal I believe to display EDMC but through Space Desk even in portrait mode for that particular screen it is so small on the screen this is currently not an option.

So I was wondering what are the chances of having a maximise option in the top right-hand corner along with the Minimize and Close buttons or the ability to resize the whole not just the current width in the App.

This would I believe enhance the great work that you are doing for the community and the immersive quality it currently gives me in the game.

I am not a coder nor do I have the IT skills to describe this idea any better than I have done so but greatly admire the work that you have done.

Trickyhicky

**Comment by Athanasius** [comment]
The paradigm of EDMC UI is that the sizing is controlled by the totality of the UI elements, i.e. core plus any addons you have with UI elements.  It's all set up to autosize depending on that, with the small amount of control over the width.

There's potential for a lot of plugins' UI to break in subtle ways if the code was changed to allow manual resizing.

Have you looked at Settings > Appearance > UI Scaling Percentage to just scale the whole UI up to suit your use case ?

**Comment by Trickyhicky** [comment]
Yeah, I tried that, but I lost the bottom of the Settings page and had to resize through the convoluted route of the RegEdit, which for me is scary. Okay, no probs it is a shame but I guess I will stick with what I have.
 I will be reviewing what plugins to keep because as you say they mess with the resizing in general and another app I use gives me the info I need in a more user-friendly way.  
Once again thankyou for all your hard work it is appreciated.
